packages/lesswrong/lib/geocoding.ts

```diff
@@ -32,3 +32,22 @@ export function pickBestReverseGeocodingResult(results: any[]) {
   return results[0];
 }
 
+/**
+ * Given a Google Maps reverse geocoding result, find the country code for that location.
+ * See documentation for data structure:
+ * https://developers.google.com/maps/documentation/geocoding/requests-reverse-geocoding#reverse-example
+ *
+ * @param googleLocation
+ * @returns {string|null}
+ */
+export function getCountryCode(googleLocation: AnyBecauseHard): string|null {
```

Comment:
Thanks, yeah I considered using that but that depends on browser local storage and I wanted to avoid the job ad loading in on the browser and shifting the layout, if possible. But if we move the ad to a location below the fold then we could change it.

Also I wonder if people would feel sketched out if we use that location data for site features like job ads. At least with the "account location" (`googleLocation`) user field, users explicitly tell us what location they want to use / explicitly opt in to using their browser location, and they can update this value in their account settings.